tf/net.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python
+
+import argparse
+import gzip
+import os
+import numpy as np
+import lc0net_pb2 as pb
+
+LC0_MAJOR = 1
+LC0_MINOR = 0
+LC0_PATCH = 0
+
+
+class Net:
+    def __init__(self):
+        self.pb = pb.Net()
+        self.pb.min_version.major = LC0_MAJOR
+        self.pb.min_version.minor = LC0_MINOR
+        self.pb.min_version.patch = LC0_PATCH
+        self.weights = []
+
+
+    def fill_layer(self, layer, weights):
+        """Normalize and populate 8bit layer in protobuf"""
+        params = np.array(weights.pop(), dtype=np.float32)
+        layer.min_val = 0 if len(params) == 1 else np.min(params)
```

Comment:
This seems to be protecting against a divide by 0, but if the single parameter is 0, then you still get a divide by 0.
Probably better to just do max and min and if max == min, max = min + 1 or something?